project-generator/snippets/imports.js

```diff
@@ -0,0 +1,42 @@
+import path from "path";
+import * as types from "@onflow/types";
+import {
+    init,
+    sendTransaction,
+    deployContractByName,
+    getTransactionCode,
+} from "flow-js-testing/dist";
+import { getScriptCode } from "flow-js-testing/dist/utils/file";
+import { executeScript } from "flow-js-testing/dist/utils/interaction";
+import { getContractAddress } from "flow-js-testing/dist/utils/contract";
+import { getAccountAddress } from "flow-js-testing/dist/utils/create-account";
+
+const basePath = path.resolve(__dirname, "../##BASE-FOLDER##");
```

Comment:
Should this file be a handlebars template?

Reply:
And this one as well.
`package.json` probably could be replaced as well 🤔